test/org/sagebionetworks/bridge/play/controllers/AuthenticationControllerMockTest.java

```diff
@@ -538,6 +545,25 @@ public void signInBlockedByVersionKillSwitch() throws Exception {
         controller.signInV3();
     }
     
+    @Test
+    public void signInOnLocalDoesNotSetCookieWithSSL() throws Exception {
+        String json = TestUtils.createJson(
+                "{'study':'" + TEST_STUDY_ID_STRING + 
+                "','email':'email@email.com','password':'bar'}");
+        response = mockPlayContextWithJson(json);
+        when(controller.bridgeConfig.getEnvironment()).thenReturn(Environment.LOCAL);
+        when(controller.bridgeConfig.get("webservices.url")).thenReturn("http://localhost:9000");
+        
+        UserSession session = createSession(null, null);
+        when(authenticationService.signIn(any(), any(), any())).thenReturn(session);
+        
+        controller.signIn();
+        
+        verify(response).setCookie(BridgeConstants.SESSION_TOKEN_HEADER, TEST_SESSION_TOKEN,
+                BridgeConstants.BRIDGE_SESSION_EXPIRE_IN_SECONDS, "/",
+                BridgeConfigFactory.getConfig().get("webservices.url"), false, false);
```

Comment:
This will fail in Travis, since we mock bridgeConfig to return localhost, but in travis, this picks up the dev config.